### ISSUE TITLE
Markdown ao invés de PDF

### ISSUE THREAD

**Issue by ajsaraujo** [gustavoguanabara/html-css#54]
Olá pessoal, primeiramente, parabéns pelo material, que está ótimo como sempre.

Gostaria de deixar uma sugestão para que ao invés de usarem PDFs, deixem os desafios em Markdown, fica muito mais fácil a visualização aqui no GitHub. 

Valeu!

**Comment by kalebzaki4** [comment]
Reforço a ideia e é bem provavel que fique melhor com Markdown!

**Comment by BenjamimCS** [comment]
Verdade. Uma pena dos módulos já terem sido todos gravados. Se a licensa permitir, nós, alunos, poderiamos fazer isso — com todos os créditos ao mestre, é claro.

**Comment by flameastro** [comment]
> Olá, @ajsaraujo  ! Você poderia fechar essa issue?

Agradeço!
